rest_api/tests/api_test/fixtures.py

```diff
@@ -132,6 +133,211 @@ def invalid_batch():
     
     return data
 
+@pytest.fixture(scope="session")
+def setup_batch_valinv_txns(request):
+    """Setup method for posting batches and returning the 
+       response
+    """
+    data = {}
+    signer = get_signer()
+    expected_trxn_ids  = []
+    expected_batch_ids = []
+    expected_trxns  = {}
+    expected_batches = []
+    initial_batch_length = batch_count()
+    initial_transaction_length = transaction_count()
+
+    LOGGER.info("Creating intkey transactions with set operations")

```

Comment:
There is a need to generalize these fixtures now....otherwise they will simply add locs